tests/Shared/WorkloadTesting/data/nuget8.config

```diff
@@ -9,7 +9,7 @@
     <add key="built-local" value="%BUILT_NUGETS_PATH%" />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
-    <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
+    <add key="dotnet9" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet9/nuget/v3/index.json" />
```

Comment:
So we don't need dotnet8 feed anymore? I think we still push our packages there so we have to update our default channels in darc.

Reply:
But we shouldn't be fetching any of our packages from these feeds (except `BUILT_NUGETS_PATH`) for tests. Or am I missing something?